index.md

```diff
@@ -7,8 +7,7 @@ listing:
   categories: true
   sort-ui: false
   filter-ui: false
+max-description-length: 250
```

Comment:
The intro to the one post we currently have was being cut off. I felt 250 was a reasonable length for a post description.